internal/commands/fmt.go

```diff
@@ -60,7 +60,7 @@ func doFmt(args []string, config *fmtCommandConfig) error {
 				return err
 			}
 		default:
-			if shouldFormat(config, dir) {
+			if shouldFormat(config, path, dir) {
```

Comment:
is there an assertion that `os.Stat` reports `path` being a file rather than a directory?
it should not be the case that `qbec alpha fmt dir.json` (where `dir.json` is a directory) is treated as a json file

Reply:
The comment may not be in the right place but you get what I mean...

Reply:
Added the test case.